pkg/yangentry/build_yang.go

```diff
@@ -12,23 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package util contains goyang utility functions that could be useful for
-// external users.
-package util
+// Package yangentry contains high-level helpers for using yang.Entry objects.
+package yangentry
 
 import (
 	"fmt"
 
 	"github.com/openconfig/goyang/pkg/yang"
 )
 
-// ProcessModules takes a list of either module/submodule names or .yang file
+// Parse takes a list of either module/submodule names or .yang file
 // paths, and a list of include paths. It runs the yang parser on the YANG
 // files by searching for them in the include paths or in the current
 // directory, returning a slice of yang.Entry pointers which represent the
 // parsed top level modules. It also returns a list of errors encountered while
 // parsing, if any.
-func ProcessModules(yangfiles, path []string) (map[string]*yang.Entry, []error) {
+func Parse(yangfiles, path []string) (map[string]*yang.Entry, []error) {
```

Comment:
Thanks for the suggestion. I think it would be `yangentry.Get` per the naming guidelines to be concise since this is less repetitive than `yangentry.GetYang`. However `Get` now seems just a little more vague than `Parse` since `Parse` also conveys that it's parsing YANG files.

Reply:
`Parse` SGTM to avoid stuttering.

Reply:
ah yes, thanks for putting into context.  yangentry.Parse has good context and semantics